osu.Game/Screens/Select/FilterCriteria.cs

```diff
@@ -13,5 +14,7 @@ public class FilterCriteria
         public string SearchText;
         public RulesetInfo Ruleset;
         public bool AllowConvertedBeatmaps;
+        public double MinimumStarDifficulty = OsuConfigManager.DEFAULT_DISPLAY_STARS_MINIMUM;
```

Comment:
make these `double?` rather than funnelling in a default value from `OsuConfigManager`. Then if they are null you can bypass the filtering.